pauthabielf64/pauthabielf64.rst

```diff
@@ -206,6 +207,14 @@ This document is at **Alpha** release quality.
   |            |                     | descriptions of existing encoding and relocations. Add           |
   |            |                     | description of SHT_AUTH_RELR.                                    |
   +------------+---------------------+------------------------------------------------------------------+
+  | 0.3        | 22nd October 2020   | Delete the appendix giving an encoding for GOT generating        |
+  |            |                     | relocations using the relocation addend to carry the signing     |
```

Comment:
double `signing signing`

Reply:
Ack. will fix, along with all the comments. If there are any review comments on the document as a whole I'll be happy to fix them in another pull request.